mitiq/shadows/rotation_gates.py

```diff
@@ -0,0 +1,68 @@
+from typing import List
+import cirq
+import numpy as np
+
+# generate N random Pauli strings for given number of qubits
+
+
+def generate_random_pauli_strings(
+    num_qubits: int, num_strings: int
+) -> List[str]:
+    """Generate a list of random Pauli strings.
+
+    Args:
+        num_qubits: The number of qubits in the Pauli strings.
+        num_strings: The number of Pauli strings to generate.
+
+    Returns:
+        A list of number of `num_strings` random Pauli strings
+          of length `num_qubits`.
```

Comment:
```suggestion
        A list of random Pauli strings.
```
Since function arguments are already explained before.